<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Authorization</class>
 <widget class="QWidget" name="Authorization">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Authorization</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>90</y>
     <width>371</width>
     <height>251</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(23, 23, 23);
border-radius: 20px;</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>371</width>
      <height>251</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <layout class="QVBoxLayout" name="layout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>20</number>
     </property>
     <property name="rightMargin">
      <number>20</number>
     </property>
     <item>
      <widget class="QLineEdit" name="inputNickname">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>70</height>
        </size>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit#inputNickname {
background-color: qlineargradient(spread:pad, x1:0.619, y1:0.334773, x2:0.278727, y2:0.761, stop:0 rgba(29, 29, 29, 255), stop:1 rgba(16, 16, 16, 255));
border-radius: 15px;


/* Текст */

width: 161px;
height: 37px;

font-family: 'Roboto';
font-style: normal;
font-weight: 700;
font-size: 22px;
line-height: 22px;

/* identical to box height */
qproperty-alignment: AlignCenter;
color: #FFFFFF;
}</string>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhNone</set>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Nickname</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="inputPassword">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>70</height>
        </size>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit#inputPassword {
background-color: qlineargradient(spread:pad, x1:0.619, y1:0.334773, x2:0.278727, y2:0.761, stop:0 rgba(29, 29, 29, 255), stop:1 rgba(16, 16, 16, 255));
border-radius: 15px;


/* Текст */

width: 161px;
height: 37px;

font-family: 'Roboto';
font-style: normal;
font-weight: 700;
font-size: 22px;
line-height: 22px;

/* identical to box height */
qproperty-alignment: AlignCenter;
color: #FFFFFF;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>Password</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>70</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QWidget" name="horizontalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>331</width>
          <height>51</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <item>
          <widget class="QPushButton" name="buttonSignin">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>45</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
	font-family: 'Roboto';
	font-style: normal;
	font-weight: 700;
	font-size: 22px;
	line-height: 22px;
	border-radius: 22px;
	color: white;
	background-color: rgb(65, 184, 77);
}
QPushButton:hover {
	background-color: rgb(60, 177, 72);
}</string>
           </property>
           <property name="text">
            <string>Sign in</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
